src/components/middle/message/Message.tsx

```diff
@@ -1429,8 +1429,9 @@ export default memo(withGlobal<OwnProps>(
     } = ownProps;
     const {
       id, chatId, viaBotId, replyToChatId, replyToMessageId, isOutgoing, forwardInfo,
-      transcriptionId, isPinned, replyToStoryUserId, replyToStoryId, repliesThreadInfo,
+      transcriptionId, isPinned, replyToStoryUserId, replyToStoryId,
     } = message;
+    const repliesThreadInfo = message.repliesThreadInfo || album?.messages[0].repliesThreadInfo;
```

Comment:
I think you should add optional chaining when reading `repliesThreadInfo` to avoid the error `cannot read properties of undefined (reading 'repliesThreadInfo')`